ReviewsReceiver/src/app.ts

```diff
@@ -0,0 +1,32 @@
+import "dotenv/config";
+import express, { Express } from "express";
+import mongoose from "mongoose";
+import cors from "cors";
+import bodyParser from "body-parser";
+import batchRouter from "./routes/batchRoutes";
+import apiKeyRouter from "./routes/apiKeyRoutes";
+
+
+const app = express();
+
+const initApp = () => {
+  app.use(cors());
+  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded({ extended: true }));
+  app.use("/batch", batchRouter);
```

Comment:
Maybe we should call it '/reviews' no? according to the REST guidelines, the best practices is to call routes with noun, just like 'keys' in the line below